src/hooks/useNft.ts

```diff
@@ -85,7 +121,7 @@ export const useNft = (tokenId: number) => {
   };
 
   watchEffect(() => {
-    fetchNftParts();
+    // fetchNftParts();
```

Comment:
Hi @bobo-k2 
Can I uncomment out this line so that the inventory of parent NFTs are displayed here? 
![image](https://user-images.githubusercontent.com/92044428/222650764-6ffcc26e-8009-41a4-81dd-9c102e3f9698.png)